airbyte-server/src/main/java/io/airbyte/server/handlers/OperationsHandler.java

```diff
@@ -173,9 +173,9 @@ public void deleteOperationsForConnection(final StandardSync standardSync, final
     for (final UUID operationId : deleteOperationIds) {
       operationIds.remove(operationId);
       boolean sharedOperation = false;
-      for (final StandardSync sync : allStandardSyncs) {
+      for (final StandardSync sync : configRepository.listStandardSyncsUsingOperation(operationId)) {
```

Comment:
I'm not able to comment on it, but line 171 should be removed from this function, since we no longer need to list all standard syncs:
```
final List<StandardSync> allStandardSyncs = configRepository.listStandardSyncs();
```
If this is not removed, we likely will not see as much of a performance improvement from this as we would like!

Reply:
Thanks, I removed the line.